chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/ComplaintListingMap.java

```diff
@@ -0,0 +1,91 @@
+package gov.healthit.chpl.domain;
+
+import java.util.Date;
+
+import org.springframework.beans.BeanUtils;
+
+import gov.healthit.chpl.dto.ComplaintListingMapDTO;
+
+public class ComplaintListingMap {
+    private Long id;
+    private Long complaintId;
+    private Long listingId;
+    private Date creationDate;
```

Comment:
Do we need creation date, last modified date, last modified user, and deleted fields in the domain object? I'm not sure others have them... maybe some but not all of those fields? If they need to be shown on the UI that's fine they are here of course.